sound/snapcast/Makefile

```diff
@@ -0,0 +1,126 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=snapcast
+PKG_VERSION:=0.28.0
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/badaix/snapcast.git
+PKG_MIRROR_HASH=288325fc8b134502adb522711c18210794734675684cf5d8eba8a0a82778f6c9
+
+PKG_MAINTAINER:=Szabolcs Hubai <szab.hu@gmail.com>, David Andreoletti <david@andreoletti.net>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+PKG_CPE_ID:=cpe:/a:badaix:snapcast
+
+PKG_BUILD_PARALLEL:=1
+
```

Comment:
```suggestion
```

parallel build is enabled by default for cmake.

Reply:
Accepted suggested change!